.env.example

```diff
@@ -182,6 +182,9 @@ LNBITS_DEFAULT_WALLET_NAME="LNbits wallet"
 # LNBITS_AD_SPACE_TITLE="Supported by"
 # csv ad space, format "<url>;<img-light>;<img-dark>, <url>;<img-light>;<img-dark>", extensions can choose to honor
 # LNBITS_AD_SPACE="https://shop.lnbits.com/;https://raw.githubusercontent.com/lnbits/lnbits/main/lnbits/static/images/lnbits-shop-light.png;https://raw.githubusercontent.com/lnbits/lnbits/main/lnbits/static/images/lnbits-shop-dark.png"
+# LNBITS_HP_ELEMENTS=true # if set to true, the ad space will be displayed on the home page
+# LNBITS_CUSTOM_BADGE="USE WITH CAUTION - LNbits wallet is still in BETA" 
+# LNBITS_CUSTOM_BADGE_COLOR="warning"
```

Comment:
This property is too fine grained, it pollutes the settings namespace.
```suggestion
```

Reply:
Its important the badge color is definable